recipient-resolver/src/main/java/com/redhat/cloud/notifications/recipientresolver/model/User.java

```diff
@@ -0,0 +1,89 @@
+package com.redhat.cloud.notifications.recipientresolver.model;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
This class contains many fields that will never be used. We'll simplify it later, after this PR is merged.